specs/vm/instruction_set.md

```diff
@@ -131,6 +131,7 @@ If the [`F_WRAPPING`](./main.md#flags) flag is set, an operation that would have
 | Operation   | ```$rA = $rB + $rC;``` |
 | Syntax      | `add $rA, $rB, $rC`    |
 | Encoding    | `0x00 rA rB rC -`      |
+| Effects     | No effects             |
```

Comment:
Perhaps we can assume "No effects" by default, and only add an `Effects` row for "impure" instructions?

Reply:
On one hand, I don't have a preference here, on the other hand, if the effects row is always present it makes the instruction summaries more uniform and makes sure we didn't forget to annotate an instruction or two :)

As you suggested in the other comment above, we should definitely enumerate the set of all possible effects in `vm/index.md` to make it uniform.

Reply:
Addressed review suggestion in a new commit, I think it's ready for review now